mindjunkies/dashboard/views.py

```
@@ -1,62 +1,98 @@
+            return redirect("verification_wait")
+        return render(request, "apply_teacher.html")
+
+
+class ContentListView(LoginRequiredMixin, View):
+    permission_required = VIEW_COURSE_PERMISSION
+
+    def get(self, request: HttpRequest, status:str) -> HttpResponse:
+        if not request.user.is_teacher:
+            return redirect("teacher_permission")
+
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)